consensus/polybft/state_sync_relayer.go

```diff
@@ -59,31 +53,28 @@ var _ StateSyncRelayer = (*stateSyncRelayerImpl)(nil)
 type stateSyncRelayerImpl struct {
```

Comment:
Should we come up with some different name? (e.g. `externalBridgeMsgRelayerImpl`)